Aliases/v8@5.1

```diff
@@ -0,0 +1 @@
+../Formula/v8.rb
```

Comment:
Why name the alias `v8@5.1` and not `v8@5` which seems more consistent with our policy?